packages/angular/cli/models/version.ts

```diff
@@ -22,8 +22,6 @@ export class Version {
   }
 }
 
-// TODO: Convert this to use build-time version stamping after flipping the build script to use bazel
-// export const VERSION = new Version('0.0.0-PLACEHOLDER');
 export const VERSION = new Version(
   (
     JSON.parse(readFileSync(resolve(__dirname, '../package.json'), 'utf-8')) as { version: string }
```

Comment:
Can't do this shortcut anymore now that there are two different version placeholders.

Reply:
Unless.... will this only apply to the `cli` package? That should never get an experimental version, right?